src/components/TextareaField/TextareaField.tsx

```diff
@@ -111,6 +116,21 @@ const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
   },
 );
 
+/**
+ * Given two lengths that may not be defined, return the smaller of the defined lengths
+ * TODO-AH: make this take a rest and can operate on any numbe of values
+ * TODO-AH: move this to utilities
+ */
+function getSmallest(lengthA?: number, lengthB?: number): number | undefined {
+  if (lengthA !== undefined && lengthB !== undefined) {
+    return Math.min(lengthA, lengthB);
+  } else if (lengthA && lengthB === undefined) {
+    return lengthA;
+  } else if (lengthB && lengthA === undefined) {
+    return lengthB;
+  }
+}
```

Comment:
Code golf time 🏌️‍♀️

```suggestion
function getSmallest(...args: Array<number | undefined>): number | undefined {
  if (args.length === 0) {
    return;
  }
  return Math.min(...args.filter(Boolean) as number[]);
}
```

or with type predicate on the filter:

```suggestion
function getSmallest(...args: Array<number | undefined>): number | undefined {
  if (args.length === 0) {
    return;
  }
  return Math.min(...args.filter((n): n is number => Boolean(n)));
}
```

Reply:
I like these ⛳! Since the other PR builds upon this one and moves this into a utility, will commit this as-is, and take some strokes off in #1771